test/edu-benefits/1990n/schema/schema.unit.spec.js

```diff
@@ -0,0 +1,46 @@
+import { expect } from 'chai';
+import fs from 'fs';
+import path from 'path';
+import { Validator } from 'jsonschema';
+
+import { transform } from '../../../../src/js/edu-benefits/1990n/helpers';
+import formConfig from '../../../../src/js/edu-benefits/1990n/config/form';
+import fullSchema1990n from 'vets-json-schema/dist/22-1990N-schema.json';
+
+describe('1990n schema tests', () => {
+  const v = new Validator();
+  const files = fs.readdirSync(__dirname);
+  let expectedDataFiles;
+  try {
+    expectedDataFiles = fs.readdirSync(path.join(__dirname, 'expected-data'));
```

Comment:
Yeah, I thought about deleting it. We probably won't need it, but I was just thinking if we ever copied the test over to somewhere that needed it...

Would you recommend deleting it? I'm on the fence.